### ISSUE TITLE
Correct text generation with GPT

### ISSUE THREAD

**Issue by eatsleepraverepeat** [kpot/keras-transformer#27]
Hello.

Following examples I've successfully trained universal transformer GPT model on custom training corpus. Validation perplexity values are good, but I'm struggling to generate readable text with the model.

By padding arbitrary input up to the maximum sequence length, sequence is passing through the model and I take logits for a next token. So, if an input was 5 tokens long, it's padded up to the MAX_SEQ_LEN (256 in my case). Then, I'm using 6-th token logits to do a **top k sampling**, replace sequence with new token and passing again.

Am doing something wrong? Is padding can be an issue?

**Comment by eatsleepraverepeat** [comment]
Found the problem.
Closing.
